src/Microsoft.Data.SqlClient/tests/ManualTests/DataCommon/DataTestUtility.cs

```diff
@@ -330,8 +325,6 @@ public static bool GetSQLServerStatusOnTDS8(string connectionString)
             return isTDS8Supported;
         }
 
-        public static bool IsNotX86Architecture => RuntimeInformation.ProcessArchitecture != Architecture.X86;
```

Comment:
this is being used in DistributedTransactionTest. Failure is due to removing this one.